docs/.gitignore

```diff
@@ -0,0 +1,32 @@
+# Dependencies
+docs/node_modules
```

Comment:
`node_modules` will ignore `docs/node_modules` and the same for the other dupes.